commands/deploy_test.go

```diff
@@ -210,6 +212,7 @@ func TestCommandsDeployUploadsTarball(t *testing.T) {
 		body      []byte
 		accountID int
 		file      []byte
+		headers   map[string]string
```

Comment:
I still lean towards us re-using the http.Header type in places like this, since it has certain useful methods already attached and shows the intent very clearly. Not a pr-breaker though.

Reply:
Good call, I have fixed this is a61f6c6.